src/compiler/diagnosticMessages.json

```diff
@@ -2513,7 +2513,7 @@
         "category": "Message",
         "code": 6019
     },
-    "Compile the project in the given directory.": {
+    "Compile the project in the given directory, using 'tsconfig.json' or the specified config file.": {

```

Comment:
how about:
`Compile the project defined by the config file. Argument can be a full path to a config file, or a directory path to a directory containing 'tsconfig.json'`